services/web/server/src/simcore_service_webserver/session/access_policies.py

```diff
@@ -1,18 +1,20 @@
 import functools
 import logging
 import time
+from collections.abc import Iterator
 from contextlib import contextmanager
-from typing import Iterator, TypedDict
+from typing import TypedDict
 
 from aiohttp import web
 from aiohttp_session import Session
 from pydantic import PositiveInt, validate_arguments
 from servicelib.aiohttp.typing_extension import Handler
 
-from .plugin import get_session
+from .api import get_session
 from .settings import SessionSettings, get_plugin_settings
 
 _SESSION_GRANTED_ACCESS_TOKENS_KEY = f"{__name__}._SESSION_GRANTED_ACCESS_TOKENS_KEY"
+_HTTP_MIN_ERROR_CODE = 400  # Errors start in this code https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
```

Comment:
Q: why not use `web.HTTPNotFound.status_code` since we tendo to use this pattern elsewhere?

Reply:
Because it is not really a not-found what I want but the "first number" for all the errors